src/plugins/intel_gpu/src/plugin/transformations/op/swiglu.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "intel_gpu/op/swiglu.hpp"
+#include "validation_util.hpp"
+#include "variadic_split_shape_inference.hpp"
+#include "openvino/core/partial_shape.hpp"
+#include "openvino/op/variadic_split.hpp"
+
+namespace ov {
+namespace intel_gpu {
+namespace op {
+
+SwiGLU::SwiGLU(const Output<Node>& data,
+               const std::vector<int64_t>& axis,
+               const std::vector<int64_t>& split_lengths,
```

Comment:
Actually, I think we do not need to limit the split length as constant, dont' we ? 
(Please leave a TODO note if it is just a left work) 
Actually from the definition of the swiglu, I think we can calculate the length from the kernel using the input length.

Reply:
Actually when doing shape inference inside swiglu internal op, shape_infer of variadict split is used, and the original constant data is needed at this time. Therefore, when creating a primitive from op, it's necessary to retain the origignal  constant data from variadic split.